README.md

```diff
@@ -759,6 +759,10 @@ Happy Coding! -->
 
 <tr align="center">
 <td align="center"> 250.</td>
+<td align="center"> <a href="https://github.com/pranjay-poddar/Dev-Geeks/tree/main/Frontend-Projects/ApppleClone"> Apple Clone </a></td>
```

Comment:
Create new project item
Do not add it in one item

Reply:
> 

Sorry will you please elaborate

Reply:
Like create new item for your project addition
Do not add new project in already added project item